models/user.js

```diff
@@ -12,6 +14,13 @@ const USER_ROLES = [
   'moderator'
 ];
 
+if (!process.env.TALK_SESSION_SECRET) {
+  debug('\n////////////////////////////////////////////////////////////\n' +
+        '///   TALK_SESSION_SECRET must be defined to encode      ///\n' +
+        '///   JSON Web Tokens and other auth functionality       ///\n' +
+        '////////////////////////////////////////////////////////////');
+}
+
```

Comment:
Should just throw an error here instead to stay consistent with the other error states of the application. Debug is also never printed unless the DEBUG env var is set.